returnn/search.py

```diff
@@ -88,7 +90,9 @@ def __init__(
 
         self.out_returnn_config_file = self.output_path("returnn.config")
 
-        self.out_search_file = self.output_path("search_out")
+        self.out_search_file = self.output_path(
+            f"search_out.{output_mode}." + (".gz" if output_gzip else "")
```

Comment:
```suggestion
            f"search_out.{output_mode}" + (".gz" if output_gzip else "")
```

Reply:
also: ```search_output``` to ```search_output.{txt,py}{,.gz}```

this change should already change the hashes of the following jobs

Reply:
Is V2 already heavy in use? Or can we change this although breaking hashes?

Reply:
I think it is. I changed this back.